threescale_api/resources.py

```diff
@@ -509,6 +509,34 @@ def __init__(self, *args, entity_name='backend_api',
     def url(self) -> str:
         return self.threescale_client.admin_api_url + '/backend_apis'
 
+    def list(self, **kwargs):
+        return list(super().list(**kwargs))
+
+    def _list(self, **kwargs):
+        if "page" in kwargs.get("params", {}):
+            return super()._list(**kwargs)
+
+        pagenum = 1
+
+        kwargs = kwargs.copy()
+        if "params" not in kwargs:
+            kwargs["params"] = {}
+
+        kwargs["params"]["page"] = pagenum
+        kwargs["params"]["per_page"] = 500
```

Comment:
I would parametrize this

Reply:
That is the max amount of items allowed by Porta.

Reply:
as mentioned 500 is max and max is optimal value for an "iterator" (unless expected to run on microcontroller).

For other (than iterator) cases it can be parametrized as `list(params={"page": 3, "per_page": 7})`